go/vt/vtgate/executor_test.go

```diff
@@ -1996,7 +1996,7 @@ func TestGetPlanUnnormalized(t *testing.T) {
 		t.Errorf("getPlan(query1): plans must be equal: %p %p", plan1, plan2)
 	}
 	want := []string{
-		query1,
+		"@unknown:" + query1,
```

Comment:
what is the effect downstream of the default being `@unknown`? Is it ok because all calls provide a valid tabletType?

Reply:
`@unknown` is actually not a valid tablet type. I didn't bother initializing the test with master or replica because that's not what I'm testing.